frontend/src/scenes/dashboard/Dashboard.js

```diff
@@ -28,8 +28,8 @@ function _Dashboard({ id }) {
                     <p>A dashboard with the ID {id} was not found!</p>
                     <HedgehogOverlay type="sad" />
                 </>
-            ) : items.length > 0 ? (
-                <DashboardItems logic={logic} />
+            ) : items && items.length > 0 ? (
+                <DashboardItems logic={logic} inSharedMode={share_token} />
```

Comment:
If `inShareMode` is expected to be a bool, it's better to write `<DashboardItems logic={logic} inSharedMode={!!share_token} />` to convert the string to a bool.

Otherwise you'll need to fix it once we migrate to TS ;).